### DISCUSSION TITLE
More display columns for SQL Server when you view the list of tables in a schema

### DISCUSSION THREAD

**Discussion by neilsteyn** [dbeaver/dbeaver#21534] [Ideas]
Oracle has all these columns
![oracle](https://github.com/dbeaver/dbeaver/assets/20437001/d5bb86d0-db88-4245-ac67-c0f06804d4bf)

but SQL Server only has these few.
![sqlsrv](https://github.com/dbeaver/dbeaver/assets/20437001/a173d9fe-fbb6-48c8-b81b-26d51a4aa12c)

what would be nice is to see the total number of rows each table has in SQL Server too

**Comment by E1izabeth** [comment]
Issue #21594
